### PR TITLE
add new setting key for retro feature

### PR DESCRIPTION
**Pull request by jayreddy519**

### CODE REVIEW

**Review by markgoho — CHANGES_REQUESTED** [review]
@jayreddy519 can we make the fallback values (`|| false`) match the value currently set?

File: config/client_config/dc/system/config/templates/features/enroll_app/hbx_admin_config.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ registry:
         - key: :drop_enrollment_members
           is_enabled: false
         - key: :drop_retro_scenario
-          is_enabled: false
+          is_enabled: <%= ENV['RETRO_FEATURE_IS_ENABLED'] || true %>
```
Comment (markgoho) [inline config/client_config/dc/system/config/templates/features/enroll_app/hbx_admin_config.yml:29]:
```suggestion
          is_enabled: <%= ENV['RETRO_FEATURE_IS_ENABLED'] || false %>
```

File: config/client_config/me/system/config/templates/features/enroll_app/hbx_admin_config.yml [hunk 2]
```diff
@@ -26,4 +26,4 @@ registry:
         - key: :drop_enrollment_members
           is_enabled: <%= ENV['DROP_ENROLLMENT_FEATURE_IS_ENABLED'] || true %>
         - key: :drop_retro_scenario
-          is_enabled: false
+          is_enabled: <%= ENV['RETRO_FEATURE_IS_ENABLED'] || true %>
```
Comment (markgoho) [inline config/client_config/me/system/config/templates/features/enroll_app/hbx_admin_config.yml:29]:
```suggestion
          is_enabled: <%= ENV['RETRO_FEATURE_IS_ENABLED'] || false %>
```

File: system/config/templates/features/enroll_app/hbx_admin_config.yml [hunk 3]
```diff
@@ -26,4 +26,4 @@ registry:
         - key: :drop_enrollment_members
           is_enabled: false
         - key: :drop_retro_scenario
-          is_enabled: false
+          is_enabled: <%= ENV['RETRO_FEATURE_IS_ENABLED'] || true %>
```
Comment (markgoho) [inline system/config/templates/features/enroll_app/hbx_admin_config.yml:29]:
```suggestion
          is_enabled: <%= ENV['RETRO_FEATURE_IS_ENABLED'] || false %>
```

**Comment by jayreddy519** [comment]
Added new pr for this change.

**Pull request closed without merging**